1228-public_page/1530-files_with_administrative_agreement_is_not_visible.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Files with Administrative Agreement is not visible
```

Comment:
Added @norrs to the reviewers as I think it might be interesting to observe and participate the process here, given that they are new to the process.